parser-typechecker/src/Unison/Codebase/Editor/Command.hs

```diff
@@ -57,8 +70,28 @@ type TypecheckingResult v =
          (Either Names0 (UF.TypecheckedUnisonFile v Ann))
 
 data Command m i v a where
+  -- Escape hatch.
   Eval :: m a -> Command m i v a
 
+  HQNameQuery
```

Comment:
These commands serve to move some of the backend logic from `HandleInput` into `HandleCommand`, so we can call into `Backend` from those commands. `HandleInput` doesn't have access to a `Codebase` and so it can't call the `Backend`.